.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "recommendations": [
+    "charliermarsh.ruff",
+    "magicstack.magicpython",
```

Comment:
This extension is abandoned since 2018, since it is embedded into VS Code natively now. I think we can remove it.

Reply:
magicpython